.cbmc-batch/jobs/aws_hash_table_init-bounded/Makefile

```diff
@@ -12,26 +12,22 @@
 # limitations under the License.
 
 ###########
-# Sufficient for coverage for bounded inputs
-# Additional coverage is given using the -unbounded version of this proof
-# 4 - 1 min
-# 8 - 2 min
-# 12 - 6 min
-MAX_TABLE_SIZE ?= 8
+# Using the new calloc model, get full coverage in about 5s on unbounded sizes!
+MAX_TABLE_SIZE ?= SIZE_MAX
```

Comment:
Why do you need to explicitly assign `SIZE_MAX` instead of just considering non-deterministic values?